upf/upf.c

```diff
@@ -195,6 +195,24 @@ vnet_upf_nat_pool_add_del (u8 * nwi_name, ip4_address_t start_addr,
   return 0;
 }
 
+void
+upf_pfcp_policer_init (upf_main_t * vm)
+{
+  policer_t policer_main;
+
+  memset (&policer_main, 0, sizeof (policer_t));
+
+  policer_main.single_rate = 1;
+  /* Tokens per period, according to police.h should be > 1000 */
+  policer_main.cir_tokens_per_period = 1 << 10;
+  /* scale is set to 10 for init, with given logic that message size is 1 */
+  policer_main.scale = 10;
+  /* we are making top limit of bucket to be high by default */
+  policer_main.current_limit = (1 << 31);
+
+  vec_add1 (vm->upf_pfcp_policer_main, policer_main);
```

Comment:
why use a vector here? that's just a single instance. Perhaps could just have a global var (which is already defined below in this commit btw)

Reply:
I've refactored PR to use singleton global QoS config that can be configurable via API or CLI
Configuration done via committed PPS and committed burst in ms.
@ivan4th @RoadRunnr please recheck the PR